rescheduler/rescheduler.go

```diff
@@ -130,10 +134,22 @@ func main() {
 	}
 }
 
-func waitForScheduled(podsBeingProcessed *podSet, pod *kube_api.Pod) {
-	// TODO(piosz): periodically check whether pod is actually scheduled
-	glog.Infof("Waiting for pod %s to be scheduled", podId(pod))
-	time.Sleep(10 * time.Minute)
+func waitForScheduled(client *kube_client.Client, podsBeingProcessed *podSet, pod *kube_api.Pod) {
+	for start := time.Now(); time.Since(start) < *podScheduledTimeout; time.Sleep(10 * time.Second) {
```

Comment:
Not sure if you already have this dependency in contrib (and if not how easy it is to import it), but it would be great if you could use:
wait.Poll(...)
from: https://github.com/kubernetes/kubernetes/blob/master/pkg/util/wait/wait.go#L158
function instead.

Also, please make the sleep smaller (like 500ms or sth like that).